CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ It is recommended that you go through
 with the technologies and the contribution process.
 
 ## Pull Request (contribution) process
-
+### Guidelines
```

Comment:
Can you please add another step between 1 & 2:
"If there is an open issue related to your PR, link your PR to it either manually or by using a keyword like "Fixes #issue_number" in the description (more information [here](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). If there is no open issue, consider creating one so it's easier to understand what problem your PR fixes."